pkgs/development/python-modules/pytest-rerunfailures/default.nix

```
@@ -2,13 +2,13 @@
 
 buildPythonPackage rec {
   pname = "pytest-rerunfailures";
-  version = "10.3";
+  version = "11.0";
```

```suggestion
  version = "11.0";
  format = "pyproject";
```

Please set an explicit format, and while we're at it we can migrate to the pyproject build.

This might require `setuptools` in `nativeBuildInputs`, see https://github.com/pytest-dev/pytest-rerunfailures/blob/11.0/pyproject.toml#L3.

---

[Done](https://github.com/NixOS/nixpkgs/compare/71b9b94fbc9006303abba3bc5c0be0ed2bc8988c..0d1bad0fbc58697bfac4babaf5816010cba72a57). (And it indeed requires `setuptools`.)